### ISSUE TITLE
WISH: Modifications required to jaws manager OPI

### ISSUE THREAD

**Issue by davidkeymer** [ISISComputingGroup/IBEX#7012]
After discussions with IS, it was decided that the drop-down box for "Experiment Type" should be removed as it was never used in SECI and unlikely to be in IBEX.  Also an extra set of parameters for a sixth jaws set should be added (the beamscraper near the sample area).

All other functionality should remain unchanged.

## Acceptance criteria 
- [ ] Drop-down box removed
- [ ] Controls for sixth jaw set added

## Notes
- Original ticket [here](https://github.com/ISISComputingGroup/IBEX/issues/6033)

**Comment by davidkeymer** [comment]
GUI [PR](https://github.com/ISISComputingGroup/ibex_gui/pull/1394)

Release Notes [PR](https://github.com/ISISComputingGroup/IBEX/pull/7016)

**Comment by rerpha** [comment]
Great, cheers
